### PR TITLE
add LMSL parameter code mapping to parm code lookup table

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
https://internal.cida.usgs.gov/jira/browse/IOW-775

Description
-----------
Added aq -> nwis parm code mapping for LMSL, 72150

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: liquibase/changeLogs/capture/data/aqToNwisParm/aqToNwisParm.tsv [hunk 1]
```diff
@@ -190,6 +190,7 @@ Elevation, GW, GUVD04|ft	72229	GUVD04	ft	false	true
 Elevation, GW, GUVD63|ft	72228	GUVD63	ft	false	true
 Elevation, GW, HILOCAL|ft	72230	HILOCAL	ft	false	true
 Elevation, GW, MSL|ft	72150	MSL	ft	true	false
+Elevation, GW, LMSL|ft	72150	LMSL	ft	true	false
```
Comment (ssoper-usgs) [inline liquibase/changeLogs/capture/data/aqToNwisParm/aqToNwisParm.tsv:193]: The `true` on the end signifies this is an `above datum` parameter, and the `false` indicates it is not a `below land surface` parameter.

**Review by orosz-usgs — APPROVED** [review]
looks good

File: liquibase/changeLogs/capture/data/aqToNwisParm/aqToNwisParm.tsv (hunk 1 above)
Comment (JanellFry-usgs) [reply liquibase/changeLogs/capture/data/aqToNwisParm/aqToNwisParm.tsv:193]: I think there was some talk about wanting any MSL's from Aquarius to become LMSL.
From the discussion it sounded like LMSL is the new good one and all MSL's will be transitioning to LMSL.
Guess it's Jim's call whether we should just keep this simple and show them exactly what's in AQ, or have us be the ones to turn all MSL's into LMSL's. ??

Comment (jkreft-usgs) [reply liquibase/changeLogs/capture/data/aqToNwisParm/aqToNwisParm.tsv:193]: right, We want everything to be LMSL (so line 192 should also be changed.)  https://help.waterdata.usgs.gov/code/alt_datum_cd_query?fmt=html

Comment (ssoper-usgs) [reply liquibase/changeLogs/capture/data/aqToNwisParm/aqToNwisParm.tsv:193]: Ok, so what I'm hearing is that we want to change line 192's datum value of `MSL` to `LMSL` as well?

**Pull request merged**